### PR TITLE
47 create a ledgers deposit account rest client

### LINKED ISSUE #11: Create a ledgers bank account rest client if not existing. e5
**Title: Implement REST Client for Ledger Bank Account API if Non-Existent**

**Description:**
Develop a REST client for the Ledger Bank Account API  if one is not currently available. This client should facilitate secure access to the API's primary endpoints, enabling interaction with account and transaction data.

**Expected Outcome:**
A fully functional REST client to interact with the Ledger Bank Account API, offering secure and reliable access to account data for further application use.

### PR DESCRIPTION
**Pull request by NkwaTambe**
Created ledgers deposit-rest-client

### CODE REVIEW

File: ledgers-deposit-account/ledgers-deposit-account-rest-client/pom.xml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    <dependencies>
+        <dependency>
+            <groupId>org.springframework</groupId>
+            <artifactId>spring-context</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.springframework.cloud</groupId>
+            <artifactId>spring-cloud-openfeign-core</artifactId>
+            <version>4.1.3</version>
+            <scope>compile</scope>
```
Comment (Arielpetit) [inline ledgers-deposit-account/ledgers-deposit-account-rest-client/pom.xml:35]: This version has to be in the parent pom not here

File: ledgers-deposit-account/ledgers-deposit-account-rest-client/pom.xml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+        </dependency>
+        <dependency>
+            <groupId>de.adorsys.ledgers.deposit.api</groupId>
+            <artifactId>ledgers-deposit-account-rest-api</artifactId>
+            <version>6.1-SNAPSHOT</version>
+            <scope>compile</scope>
```
Comment (Arielpetit) [inline ledgers-deposit-account/ledgers-deposit-account-rest-client/pom.xml:41]: The version should not be declared like this for the module dependency, use this 
```
<version>${project.version}</version>
```

**Review by Arielpetit — CHANGES_REQUESTED** [review]

**Review by Arielpetit — CHANGES_REQUESTED** [review]
Don't you also have to write a test for this ?

File: ledgers-deposit-account/ledgers-deposit-account-rest-client/pom.xml (hunk 1 above)
Comment (NkwaTambe) [reply ledgers-deposit-account/ledgers-deposit-account-rest-client/pom.xml:35]: yes.

File: ledgers-deposit-account/ledgers-deposit-account-rest-client/pom.xml (hunk 2 above)
Comment (NkwaTambe) [reply ledgers-deposit-account/ledgers-deposit-account-rest-client/pom.xml:41]: yeah yeah

**Comment by NkwaTambe** [comment]
> Don't you also have to write a test for this ?

we won't be writting a test for these  because we won't be making use of any mappers.. test could be written if we made use of the mapper..

taking reference from `ledgers-middleware-rest-client`

**Comment by francis-pouatcha** [comment]
Changed deployment strategy

**Pull request closed without merging**